apps/app/src/plans/fiches/update-fiche/moyens/components/EditableSection.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, VisibleWhen } from '@/ui';
+import { ReactNode } from 'react';
+
+type EditableSectionProps = {
+  label: string;
+  isReadonly?: boolean;
+  hasContent: boolean;
+  onEdit?: () => void;
+  editButtonTitle?: string;
+  emptyText?: string;
+  children: ReactNode;
+};
+
+export const EditableSection = ({
+  label,
+  isReadonly = true,
+  hasContent,
+  onEdit,
+  editButtonTitle,
+  children,
+}: EditableSectionProps) => {
+  return (
+    <div className="flex flex-col gap-2">
+      <div className="flex justify-between items-center">
+        <div>
```

Comment:
```suggestion
        <div className="flex gap-2 text-sm leading-7">
```

Bizarre d'avoir besoin de mettre `leading-7` ici d'ailleurs, c'est quoi le usecase ?